pkg/scheduler/framework/types.go

```diff
@@ -98,25 +75,96 @@ const (
 	WildCard              GVK = "*"
 )
 
+// AssignedPodAdd is the event when a pod is added that causes pods with matching affinity terms
+// to be more schedulable.
+var AssignedPodAdd = ClusterEvent{Resource: Pod, ActionType: Add, Label: "AssignedPodAdd"}
```

Comment:
could you format the variables to be:

```go
var (
    // comment for foo
    foo = 
    // comment for bar
    bar = 
)
```

And no need to add an empty line in between.